scripts/bindings/bindings.go

```diff
@@ -62,17 +80,36 @@ func generate(address string, contractFPath string) {
 		log.Fatal(err)
 	}
 
-	if err := ioutil.WriteFile(contractFPath, []byte(src[0].SourceCode), os.ModePerm); err != nil {
+	var contractFiles = []string{contractFPath}
```

Comment:
Let's extract the solc detection and  download into a separate func to make this one shorter and keep functions with single-purpose scope.

Reply:
Done!